src/pipe/modules/i-raw/main.cc

```diff
@@ -314,7 +314,11 @@ void modify_roi_out(
   mod->img_param.whitebalance[3] /= mod->img_param.whitebalance[1];
   mod->img_param.whitebalance[1] = 1.0f;
 
+#ifndef __APPLE__
```

Comment:
really that doesn't exist? then maybe another wrapper in core.h like `dt_isnan`

Reply:
Yes... Too many things are not in Macos... Painful work..

Reply:
i think this patch is obsolete, the test==test performs nan testing without api call.